app/src/main/java/com/github/epfl/meili/map/MapActivity.kt

```diff
@@ -81,15 +81,23 @@ class MapActivity : AppCompatActivity(), OnMapReadyCallback {
         } else {
             getLocationPermission()
         }
+
+        //map.setOnPoiClickListener(this)
     }
 
+    /*override fun onPoiClick(poi: PointOfInterest) {
```

Comment:
Why is this code commented out?

Reply:
cf comment on activity_main.xml

Reply:
Because I can't figure out how to test it and I want to merge